### PR TITLE
[DOCS] Prebuilt rules prerequisites: Add new section to What's New in 8.3

### PR DESCRIPTION
**Pull request by joepeeples**
Contributes to #2015. Adds a new section to the What's New page in the 8.3 docs for the prebuilt rules prerequisites fields (**Related integrations**, **Required fields**, **Setup guide**).

⚠️ ***Depends on PR #2069!*** ⚠️
- [x] Wait until the changes in PR #2069 are backported & merged to `8.3`, then update link to point to the #rule-prerequisite section.

We have to update What's New in this separate PR because `whats-new.asciidoc` in `main` is currently being worked on for the 8.4 release, so I can't update `main` and then backport to `8.3` like all the other changes in PR #2069. This PR will make changes directly to the `8.3` version of `whats-new.asciidoc`.

----

Preview: [What's New | Detection rules enhancements](https://security-docs_2327.docs-preview.app.elstc.co/guide/en/security/8.3/whats-new.html#_detection_rules_enhancements) - Scroll down to "Prerequisites and setup guides for Elastic prebuilt rules"

### CODE REVIEW

File: docs/whats-new.asciidoc [hunk 1]
```diff
@@ -95,6 +95,17 @@ image::whats-new/images/8.3/actions-icon.png[]
 
 15 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added in 8.3.0.
 
+*Prerequisites and setup guides for Elastic prebuilt rules*
+// The link below will need to be updated once PR elastic/security-docs#2069 is backported to 8.3 and merged (it will create the #rule-requisites section anchor). Until then, this will omit the section reference to allow CI to pass and create a preview.
+// Full path should be: {security-guide}/rules-ui-management.html#rule-prerequisites[prerequisites]
+
+Elastic prebuilt rules now provide additional information to help you identify, install, and configure a rule's {security-guide}/rules-ui-management.html[prerequisites]. You can confirm these requirements in the *Related integrations* and *Required fields* fields on a rule's details page, and consult the rule's *Setup guide* for additional guidance.
```
Comment (benironside) [inline docs/whats-new.asciidoc:102]:
```suggestion
Elastic prebuilt rules now provide additional information to help you identify and meet their {security-guide}/rules-ui-management.html[prerequisites]. You can confirm these requirements in the *Related integrations* and *Required fields* sections of a rule's details page, and consult its *Setup guide* for additional guidance.
```

**Review by benironside — COMMENTED** [review]
Looks good, left one suggestion

**Review by nastasha-solomon — APPROVED** [review]

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Pull request merged**